ui/sagas/index.js

```diff
@@ -1,9 +1,11 @@
 import { takeLatest } from 'redux-saga/effects';
 
-import { types } from '../ducks/scripts';
+import { FETCH_SCRIPTS } from '../ducks/scripts';
```

Comment:
I'm ok with exporting the types directly (it's better for ESLint) but did you forget to do that in the two ducks files?